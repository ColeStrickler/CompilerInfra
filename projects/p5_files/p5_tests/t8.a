b:bool;

g : () -> int {
	return b;
}
