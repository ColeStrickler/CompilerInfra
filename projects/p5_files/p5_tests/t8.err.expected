FATAL [4,9]-[4,10]: Bad return value
Type Analysis Failed
